client/src/pages/NewReport/ApplicationSection/ApplicationSection.tsx

```diff
@@ -0,0 +1,151 @@
+import SectionComponent from 'components/SectionComponent/SectionComponent.tsx';
+import Table, { TableColumn } from 'components/Table/Table.tsx';
+import { useEffect, useState } from 'react';
+import TagButton from 'components/TagButton/TagButton.tsx';
+import SVGIcon from 'components/SVGIcon/SVGIcon.tsx';
+import ActionButton, { ActionButtonColor } from 'components/ActionButton/ActionButton.tsx';
+import OverlayComponent from 'components/OverlayComponent/OverlayComponent.tsx';
+import LinkComponent from 'components/LinkComponent/LinkComponent.tsx';
+import CustomPrompt from 'components/CustomPrompt/CustomPrompt.tsx';
+import { ManagmentServiceApiInstance } from 'api/managment-service';
+
+interface ApplicationDataRow {
+    id: string;
+    name: string;
+    accuracy: 'HIGH' | 'MEDIUM' | 'LOW';
+    customPrompt: string;
+    updated: string;
+    added: string;
+    [key: string]: string;
+}
+
+const ApplicationSection = () => {
+    const [rows, setRows] = useState<ApplicationDataRow[]>([]);
+    const [loading, setLoading] = useState(true);
+    const [showModal, setShowModal] = useState(false);
+
+    const fetchApplications = async () => {
+        setLoading(true);
+        try {
+            const applicationsData = await ManagmentServiceApiInstance.getApplications();
+
+            const applicationsRows = applicationsData.map(
+                (application): ApplicationDataRow => ({
+                    id: application.id,
+                    name: application.name,
+                    accuracy: application.accuracy,
+                    customPrompt: application.customPrompt,
+                    updated: application.updated,
+                    added: application.added,
+                }),
+            );
+
+            setRows(applicationsRows);
+        } catch (e: unknown) {
+            console.error('Failed to fetch applications', e);
+        } finally {
+            setLoading(false);
+        }
+    };
+
+    useEffect(() => {
+        fetchApplications();
+    }, []);
+
+    const handleAccuracyChange = (id: string, accuracy: 'HIGH' | 'MEDIUM' | 'LOW') => {
```

Comment:
Consider creating type like ReportAccuracyLevel  in management-service.ts and using this type in all related places